client/multi-currency/multi-currency-settings/enabled-currencies-list/style.scss

```diff
@@ -196,21 +196,6 @@
 
 .wcpay-confirmation-modal.wcpay-confirmation-modal.enabled-currency-delete-modal {
 	max-width: 495px;
-	.components-modal__header {
```

Comment:
Removed these styles, which are not necessary anymore and deviate from the other modals/dialogs.